File: 110a.py
from itertools import count


a = input()
l = len(a)
con=0
count_ = int(0)
if l == 2 and a=='47' or a=='74':
    print("YES")
if l == 2 and not a=='47' or a=='74':
    print("NO")
for i in a:
    if i=='4' or i=='7':
        con=1
        count_= count_ + 1;
if l>2:
 for i in a:
    if i=='4' or i=='7':
        con=1
    else:
        con=0
        count_ = str(count_)
        # print(count_)
        break
    
    
 if con==1:
      print("YES")
      
 else:
     for i in count_:
         if i == '4' or i == '7':
            con  = 1
         else :
             con = 0
            #  print(con)
             break
     if con == 1 :
         print("YES")
     else :
         print("NO")
if l<=1:
    print("NO")
      

File: 110A.cpp
#include <iostream>
#include <cmath>
#include <string>

using namespace std;

int main()
{

    string n, count2;
    int con = 0, count = 0, con7 = 0, con4 = 0;
    cin >> n;
    int l = n.length();

    for (int i = 0; i < l; i++)
    {
        if (n[i] == '7' || n[i] == '4')
        {
            con = 1;
            if (n[i] == '7')
                con7 = 1;
            else if (n[i] == '4')
                con4 = 1;
        }
        else
        {
            con = 0;
            // cout<<"Loop Testing";
            
            for (int j = 0; j < l; j++)
            {
                if (n[j] == '7' || n[j] == '4')
                {
                    count++;
                    cout<<"count:"<<count<<endl;
                    if (n[j] == '7')
                        con7 = 1;
                    else if (n[j] == '4')
                        con4 = 1;
                }
            }
            count2 = to_string(count);
            cout<<count2<<endl;
            for (int i = 0; i < l; i++)
            {   
                if (count2[i] == '7' || count2[i] == '4')
                {
                    con = 1;
                }
                else
                {
                    con = 0;
                    break;
                }
            }
            break;
        }
        
    }
    cout<<con<<con4<<con7;

    if (con == 1 && con4 == 1 && con7 == 1)
        cout << "YES" << endl;
    else
        cout << "NO" << endl;

    return 0;
}